src/librustc_back/target/i686_unknown_linux_musl.rs

```diff
@@ -17,6 +17,20 @@ pub fn target() -> TargetResult {
     base.pre_link_args.push("-m32".to_string());
     base.pre_link_args.push("-Wl,-melf_i386".to_string());
 
+    // The unwinder used by i686-unknown-linux-musl, the LLVM libunwind
+    // implementation, apparently relies on frame pointers existing... somehow.
+    // It's not clear to me why nor where this dependency is introduced, but the
+    // test suite doesn't pass with frame pointers eliminated and it passes with
```

Comment:
This sentence contradicts itself by saying test passes and does not pass when the same condition applies.